engine/dlib/src/dmsdk/dlib/hash.h

```diff
@@ -84,6 +84,7 @@ DM_DLLEXPORT uint64_t dmHashBuffer64(const void* buffer, uint32_t buffer_len);
  * @return hash [type:uint32_t] hash value
  */
 DM_DLLEXPORT uint32_t dmHashString32(const char* string);
+#define dmHashLiteral32(string) dmHashBuffer32((string), sizeof((string)) - 1)
```

Comment:
The PR title is wrong, as it isn't a function, but a macro.

Reply:
Also, our code style says you should use CAPS for macros: `DM_HASH_LITERAL_32` (which arguably isn't as good looking, but we should try to stay consistent.